types/dragino/lsn50v2-d23/adc.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$id": "https://akenza.io/dragino/lsn50v2-d23/energy.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Energy",
```

Comment:
is there a specific reason for this to be called energy as it also contains other things, wouldn't default be more adequate?